Modules/Crm/Services/UserClientService.php

```
@@ -31,4 +31,78 @@ public function getAllPaginated(int $page = 0): \Illuminate\Contracts\Pagination
+    }
+
+    /**
+     * Set all clients for a user.
+     *
+     * @param array $userIds Array of user IDs
+     *
+     * @return void
+     *
+     * @legacy-function setAllClientsUser
+     */
+    public function setAllClientsUser(array $userIds): void
```

PHPMD (2.15.0)

Avoid unused parameters such as '$userIds'. (undefined)

(UnusedFormalParameter)